code/modules/modular_computers/file_system/programs/research/ai_restorer.dm

```diff
@@ -73,7 +73,7 @@
 	// If the AI is dead, revive it.
 	if (A.stat == DEAD && !A.should_be_dead())
 		A.set_stat(CONSCIOUS)
-		A.lying = 0
+		A.set_posture(/decl/posture/standing)
```

Comment:
other postures were removed from silicons, either they should be readded or this line should be removed imo?